static/css/blog-docs.less

```diff
@@ -414,6 +414,56 @@
     - cfgov-blog-docs
 */
 
+@gutter__em: unit(@grid_gutter-width / @base-font-size-px, em);
+@gutter-half__em: @gutter__em/2;
+
+.expandable__filters {
+    border: 1px solid @gray-50;
+    margin: unit(45px / @base-font-size-px, em) 0 @gutter__em 0;
+
+    &.expandable__padded, .expandable_content {
```

Comment:
It looks like `.expandable_content` is a child of the `.expandable` and that `.expandable__filters`and `.expandable__padded` are modifiers.